packages/cli/src/cmds/dev/handler.ts

```diff
@@ -143,13 +144,40 @@ export async function devHandler(args: IDevArgs & IGlobalArgs): Promise<void> {
     const controller = new AbortController();
     onGracefulShutdownCbs.push(async () => controller.abort());
 
+    const pubkeys: PublicKey[] = [];
+    for (let i = 0; i < secretKeys.length; i++) {
+      pubkeys.push(secretKeys[i].toPublicKey());
+    }
+
+    let signers: Signers;
```

Comment:
You should move this logic into a function to prevent having to use a mutable variable. See as example https://github.com/ChainSafe/lodestar/blob/f5d6f9f2e749f21514316b26520b50c3e4d333a9/packages/cli/src/cmds/validator/keys.ts?_pjax=%23js-repo-pjax-container%2C%20div%5Bitemtype%3D%22http%3A%2F%2Fschema.org%2FSoftwareSourceCode%22%5D%20main%2C%20%5Bdata-pjax-container%5D#L16

Reply:
Done. https://github.com/ChainSafe/lodestar/pull/3502/commits/704f9cd595ce992d3cc296fbc452cad03855ab80